packages/card/index.tsx

```diff
@@ -34,6 +37,14 @@ function Card(props: CardProps) {
             {meta && meta.length > 0 ? <Meta items={meta} /> : null}
             <Heading>{heading}</Heading>
             {description ? <Description>{description}</Description> : null}
+            {productBadges &&
+            productBadges?.badges &&
+            productBadges?.badges?.length > 0 ? (
+              <ProductBadges
+                badges={productBadges.badges}
+                appearance={appearance}
```

Comment:
Maybe this is expected, but noticed the badge color is the same as the cards hover background color when appearance is dark.

<img width="960" alt="CleanShot 2022-10-05 at 17 23 11@2x" src="https://user-images.githubusercontent.com/825855/194166697-1b41f489-a1fc-4bf5-81ae-a53b878288eb.png">

Reply:
Let me double check, but I'm not sure this is accounted for in either the `Card` or `ProductBadge` spec. 😕